src/main/java/io/github/apace100/origins/power/factory/condition/PlayerConditions.java

```diff
@@ -172,6 +178,45 @@ public static void register() {
                 }
                 return ((Comparison)data.get("comparison")).compare(attrValue, data.getDouble("compare_to"));
             }));
+        register(new ConditionFactory<>(Origins.identifier("scoreboard"), new SerializableData()
+            .add("objective", SerializableDataType.STRING)
+            .add("comparison", SerializableDataType.COMPARISON)
+            .add("compare_to", SerializableDataType.INT),
+            (data, player) -> {
+                Scoreboard scoreboard = player.getScoreboard();
+                ScoreboardObjective objective = scoreboard.getObjective(data.getString("objective"));
+                String playerName = player.getName().asString();
+
+                if (scoreboard.playerHasObjective(playerName, objective)) {
+                    int value = scoreboard.getPlayerScore(playerName, objective).getScore();
+                    return ((Comparison)data.get("comparison")).compare(value, data.getInt("compare_to"));
+                } else {
+                    return false;
+                }
+            }));
+        register(new ConditionFactory<>(Origins.identifier("command"), new SerializableData()
+            .add("command", SerializableDataType.STRING)
+            .add("permission_level", SerializableDataType.INT, 4)
+            .add("comparison", SerializableDataType.COMPARISON)
+            .add("compare_to", SerializableDataType.INT),
+            (data, player) -> {
+                MinecraftServer server = player.world.getServer();
+                if(server != null) {
+                    ServerCommandSource source = new ServerCommandSource(
+                        CommandOutput.DUMMY,
+                        player.getPos(),
+                        player.getRotationClient(),
+                        player.world instanceof ServerWorld ? (ServerWorld)player.world : null,
+                        data.getInt("permission_level"),
+                        player.getName().getString(),
+                        player.getDisplayName(),
+                        player.world.getServer(),
+                        player);
+                    int output = server.getCommandManager().execute(source, data.getString("command"));
+                    return ((Comparison)data.get("comparison")).compare(output, data.getDouble("compare_to"));
```

Comment:
The "compare_to" parameter is an integer, so this needs to be changed to `data.getInt("compare_to")`